source/_integrations/sensibo.markdown

```diff
@@ -45,127 +45,211 @@ then in the Sensibo app log you will be able to distinguish between actions
 done in the app and actions done by Home Assistant.
 {% endtip %}
 
+## Supported device
+
+The **Sensibo** {% term integration %} supports the following devices and accessories.
+
+- **Sensibo Sky**: Smart AC control device.
+- **Sensibo Air**: Smart AC control device.
+- **Sensibo Air Pro**: Smart AC control device with air quality monitoring.
+- **Sensibo Pure**: Smart air purifier.
+- **Sensibo Elements**: Smart air quality monitoring.
+- **Sensibo Room Sensor**: Motion sensor and temperature readings (needs to be connected with an Air device).
+
 {% include integrations/config_flow.md %}
 
-## Binary sensors
+{% configuration_basic %}
+API key:
+  description: The previously created API key.
+{% endconfiguration_basic %}
+
+## Data fetching and limitations
+
+Data is polled from the **Sensibo** API once every minute for all devices.
+
+If polling cannot happen because of no connectivity or a malfunctioning API, it will retry a few times before failing.
```

Comment:
```suggestion
If {% term polling %}  cannot happen because of no connectivity or a malfunctioning API, it will retry a few times before failing.
```